CONTRIBUTING.md

```diff
@@ -67,15 +66,74 @@ First, we use oauth so you need to ensure that the service supports it.
 
 Remember that there are several procedures in our db to replicate.
 
+The steps to do so are:
+
+- Set the necesary vercel env vars
+  > usually `SERVICE_CLIENT_SECRET` and `SERVICE_CLIENT_ID`
+- Create the table in our DB
+
+  > **This is sketch of a possible new table**
+  >
+  > _Add any other required columns as needed_
+
+```sql
+CREATE TABLE serviceName (
+   id INT PRIMARY KEY,
+   name VARCHAR(255),
+   email VARCHAR(255),
+   updated_at DATETIME DEFAULT GETDATE() NOT NULL,
+   created_at DATETIME DEFAULT GETDATE() NOT NULL,
+   access_token VARCHAR(255),
+   refresh_token VARCHAR(255),
+   avatar_url VARCHAR(255),
+   workspace VARCHAR(255),
+   workspace_image VARCHAR(255),
+   watermelon_user VARCHAR(255),
+   deleted BIT DEFAULT 0 NULL,
+   deleted_at DATETIME DEFAULT GETDATE() NULL,
+   FOREIGN KEY (watermelon_user) REFERENCES watermelon.dbo.users(id)
+   );
+```
+
+- Edit the userSettings table
+
+  ```sql
+  ALTER TABLE userSettings ADD ServiceReadableName INT DEFAULT 3;`
+  ```
+
+  ```sql
+  UPDATE userSettings SET ServiceReadableName = 3 WHERE ServiceReadableName IS NULL;
+  ```
+
+- Create the necessary procedures
```

Comment:
The outlined steps look beautiful but this one needs to be expanded. 

Is there a format the newly created procedure should follow? 

And also, I'm lookiing at the stored procedures on Azure right now. Where's the Asana stored procedure? This confuses me. 

Please clarify these 2 things and we're good to go.

Reply:
Asana is a different PR, not yet ready, but exposed.